docs/tutorial/tutorial-5/android.rst

```diff
@@ -2,9 +2,382 @@
 Tutorial 5 - Taking it mobile: Android
 ======================================
 
-To deploy our applicaton to Android...
+The process of deploying an application to Android is very similar to the
+process for deploying as a desktop application. There is one additional
+pre-requisite: Java 8.
+
+Setting up Java 8
+=================
+
+To use BeeWare's Android tooling, you must install a Java 8 compiler and provide
+the path to it in the ``JAVA_HOME`` environment variable. This is because
+BeeWare's Android tooling depends on the Android SDK, and the Android SDK
+depends on the Java runtime and compiler being available. Some tools within
+the Android SDK specifically require version 8 of Java, so we stick to that
+in this documentation.
+
+Download & install Java 8
+-------------------------
+
+Visit the `AdoptOpenJDK website <https://adoptopenjdk.net/>`_ and
+download OpenJDK 8.
+
+.. tabs::
+
+  .. group-tab:: macOS
+
+    On macOS, the default format is a *.pkg* installer. Download and execute the
+    installer, accepting the defaults. In your terminal, run:
+
+    ``export JAVA_HOME="$(/usr/libexec/java_home -v 1.8)"``
+
+    You will need to run this every time you start a terminal session, or you
+    can add it to your `shell configuration <https://apple.stackexchange.com/a/356455>`_
+    or `launchd configuration <http://www.dowdandassociates.com/blog/content/howto-set-an-environment-variable-in-mac-os-x-launchd-plist/>`_.
+
+  .. group-tab:: Linux
+
+    On Linux, the default format is a *.tar.gz* file. Download it and unpack it to a
+    directory of your choosing. Set the *JAVA_HOME* environment variable to the
+    directory containing *bin*; for example, if you unpacked the *.tar.gz* file
```

Comment:
`JAVA_HOME` and `bin` should be double-backticks (inline quoted). Also "containing bin" - containing *which* bin?

Reply:
Aesthetically, I find a paragraph with lots of stuff in double backticks looks kind of scary and intense, so I chose italics. I'll migrate to double backticks, but you might end up agreeing with the asterisks (italics) once you see it rendered. I'll defer to your preference here!